controller_manager/src/controller_manager.cpp

```diff
@@ -313,7 +313,7 @@ controller_interface::ControllerInterfaceBaseSharedPtr ControllerManager::load_c
       controller = chainable_loader_->createSharedInstance(controller_type);
     }
   }
-  catch (const pluginlib::CreateClassException e)
+  catch (const pluginlib::CreateClassException& e)
```

Comment:
```suggestion
  catch (const pluginlib::CreateClassException & e)
```
we keep a demilitarized zone around the `&` ;)